src/web/mjs/connectors/Kanjiku.mjs

```diff
@@ -1,12 +1,65 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
 
-export default class Kanjiku extends WordPressMadara {
+export default class Kanjiku extends Connector {
 
     constructor() {
         super();
         super.id = 'kanjiku';
         super.label = 'Kanjiku';
         this.tags = [ 'manga', 'webtoon', 'german' ];
         this.url = 'https://kanjiku.net';
+		this.requestOptions.headers.set('x-cookie', 'clicked=true');
     }
-}
\ No newline at end of file
+	
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.container h1.manga_page_title');
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/mangas', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.manga_flex-row a.manga_box');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.tab1 div.manga_overview_box a.latest_ch_number');
+        return data.map(element => {
+            let payload = {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url).substring(0, this.getRootRelativeOrAbsoluteLink(element, request.url).length -1)+'0',
+                title: element.text,
+            };
+			return payload;
+        });
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(this.url + chapter.id, this.requestOptions);
+		let data = await this.fetchDOM(request, 'div.container source');
+        return data.map(element => {
+            let payload = {
```

Comment:
Can't image links be used directly instead of creating a connector URI?
I tested some on the website and none of them required a referer...